File: 백준/Bronze/2587. 대표값2/대표값2.cc
#include <algorithm>
#include <iostream>

using namespace std;

int main(void)
{
    int arr[5];
    int a;
    for(int i = 0;i < 5;i++) {
        cin >> arr[i];
    }
    sort(&arr[0], &arr[5]);
    for(int i = 0; i < 5; i++) {
        a +=arr[i];
    }
    a = a/5;
    cout << a << endl << arr[2];
}

File: 백준/Bronze/2587. 대표값2/대표값2.js
/**
 * Input
 */
const filePath = process.platform === "linux" ? "/dev/stdin" : "./src/input.txt";
const input = require("fs").readFileSync(filePath).toString().trim().split("\n");

let n = input.length;
let arr = [];
for (let i = 0;i < n; ++i) {
    arr.push(+input[i]);
}
/**
 * Solution
 */
let sum = arr.reduce((prev, cur) => prev + cur, 0);
console.log(sum / n);
arr.sort();
let mid = (n / 2).toFixed(0);
console.log(arr[mid - 1]);
